cedar-policy/src/tests.rs

```diff
@@ -3258,3 +3258,56 @@ mod error_source_tests {
         }
     }
 }
+
+mod issue_604 {
+    use crate::Policy;
+    fn to_json_is_ok(text: &str) {
+        let policy = Policy::parse(None, text).unwrap();
+        let json = policy.to_json();
+        assert!(json.is_ok());
```

Comment:
This `assert_matches` will give a better error message if this test ever fails -- instead of just something like "assert failed: json.is_ok()", it will show what the value of `json` actually was (what `Err`)

```suggestion
        assert_matches!(json, Ok(_));
```